espressif/vscode-esp-idf-extension
# Enable/Disable status bar icons in Commands Tree view

## Description

Add Checkbox in Espressif tab Commands to be enable/disable in status bar.

![Screenshot 2024-09-26 at 16 15 35](https://github.com/user-attachments/assets/c29809d1-d8d5-41db-9265-d3b4b87e769c)


Fixes #1307

## Type of change

- New feature (non-breaking change which adds functionality)

## Steps to test this pull request

Provide a list of steps to test changes in this PR and required output

1. Click on "Espressif" icon tab. You can see a list of commands with checkbox. If you click the checkbox to disable it, the icon in the status bar will disappear. The same for the opposite.

- Expected behaviour:
Icons appear and disappear from status bar based on user selection.

- Expected output:

## How has this been tested?

Manual testing

## Checklist
